lumber-generate.js

```diff
@@ -19,6 +20,23 @@ program
   .parse(process.argv);
 
 (async () => {
+  // NOTICE: Check deprecated environments variables.
```

Comment:
In line with what Arnaud told me in past reviews, Forest prefers to avoid comments that feel redundant with the code. Here, you could name the `EnvironmentChecker` `EnvironmentDeprecationChecker` and it would be obvious why this is used for :-)